ServerlessLibraryAPI/Controllers/MetricsController.cs

```diff
@@ -9,13 +9,13 @@ namespace ServerlessLibrary.Controllers
     [ApiController]
     public class MetricsController : ControllerBase
     {
-        // PUT api/<controller>/
+        // PUT api/<controller>/{userAction}
         [ProducesResponseType(typeof(bool), 200)]
-        [HttpPut()]
-        public JsonResult Put([FromBody]string template)
+        [HttpPut]
+        [Route("{userAction}")]
```

Comment:
why do you need to add this additional route attrib?
PUT on /api/metrics should work without this additional route attrib

Reply:
yes. In last iteration, this was used. Now no longer required. will remove

Reply:
removed